docker-cogstack/compose-ymls/cogstack-clust/docker-compose.yml

```diff
@@ -80,21 +92,49 @@ services:
         hard: -1
     volumes:
       - esdata3:/usr/share/elasticsearch/data
+      - ./elasticsearch/config/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml:ro
     networks:
       - esnet
 
 
+#---------------------------------------------------------------------------#
+# Kibana webapp                                                             #
+#---------------------------------------------------------------------------#
   kibana:
     image: docker.elastic.co/kibana/kibana:6.2.4
+    container_name: kibana
     ports:
-      - 5601:5601
+#      - "5601:5601"    #without proxy, forward to host:5601
+      - "5601"        #with nginx proxy forwarding host:5601/ --> [kibana]:5601 
```

Comment:
No need of this i think since on the same network as nginx.

Reply:
hmm typo I think, just though I'd leave a commented out version in case people wanted to play without the basic_auth. Then they can just hit the host ports

Reply:
Oh okay. I meant just even removing the line ` - "5601"        #with nginx proxy forwarding host:5601/ --> [kibana]:5601` since port 5601 should already be `EXPOSED` in kibana Dockerfile. Same applies for elastic port 9200.

Reply:
True the ports are already exposed, I'm not sure of convention, and it's not very DRY, but I like having this in the docker-compose.yml file so it the intent is clear without having to inspect the image or Dockerfile.